config.js

```diff
@@ -24,5 +26,20 @@ module.exports = {
 		templates: "client/hbs",
 		upload: "upload"
 	},
+	categories:[
```

Comment:
The implementation is tag-like: just add it to a post.  The first time a post is filed under a category, the category is "created".